src/frontend/initializeNodejs.ts

```diff
@@ -0,0 +1,23 @@
+import {getRandomBytes} from 'expo-crypto';
+import {getItemAsync, setItemAsync} from 'expo-secure-store';
+import nodejs from 'nodejs-mobile-react-native';
+import {uint8ArrayToHex} from 'uint8array-extras';
+
+const ROOT_KEY = '__RootKey';
+
+export async function initializeNodejs() {
+  let rootKey = await getItemAsync(ROOT_KEY);
+  if (!rootKey) {
+    try {
+      const newRootKey = uint8ArrayToHex(getRandomBytes(16));
+      await setItemAsync(ROOT_KEY, newRootKey);
+      rootKey = newRootKey;
+    } catch (err) {
+      throw new Error(
+        `Error initializing root key: ${typeof err === 'string' ? err : ''}`,
+      );
+    }
+  }
+
+  nodejs.startWithArgs(`loader.js --rootKey ${rootKey}`);
```

Comment:
I think the `=` is needed in order to appease node's `parseArgs()` without additional configuration (tried it out in the repl and ran into issues without it)

```suggestion
  nodejs.startWithArgs(`loader.js --rootKey=${rootKey}`);
```

Reply:
https://github.com/digidem/CoMapeo-mobile/pull/89/commits/1a9a0ef294e7d4226dc3159d35cf89c0a973022e